templates/default/web_app.conf.erb

```diff
@@ -3,10 +3,10 @@
 #  Local modifications will be overwritten by Chef.
 #
 <VirtualHost *:<%= node['stash']['apache2']['port'] %>>
-	<% unless node['stash']['apache2']['virtual_host_name'].empty? -%>
```

Comment:
For some reason, DigitalOcean images don't have an FQDN (from which this is set in attributes), so it's nil and there's not `#empty?`. Realized we probably shouldn't be using this method anyhow, so swapped it out.

If we really want something robust (I'm not sure it's an edgecase worth pandering to) then we could perhaps use `#blank?` from chef-sugar cookbook:
https://github.com/sethvargo/chef-sugar#core-extensions

I don't think that's worth it though -- a blank string shouldn't be set for any of these anyhow.